### DISCUSSION TITLE
Different views

### DISCUSSION THREAD

**Discussion by hunter99507** [Radiokot/photoprism-android-client#103] [Ideas]
It would be great if we could change the default view of the pictures. Even changing the style would be great like a mosaic type view and size.

**Comment by Radiokot** [comment]
Hi. I agree, the ability to change tile size is in the project backlog and will be added eventually.

**Comment by Radiokot** [comment]
The ability to change the item size is added to the preferences in [1.24.0](https://github.com/Radiokot/photoprism-android-client/releases/tag/1.24.0(37)).
Pinch gesture will be added later.
